web/client/translations/data.it-IT.json

```diff
@@ -118,6 +118,17 @@
                 "noVendor": "No Vendor",
                 "geoserver": "GeoServer"
             },
+            "availableTileGrids": "Tile grids disponibili",
+            "crsId": "Id",
+            "projection": "Proiezione",
+            "tileSize": "Dimensione tile",
+            "tileGridInUse": "Il tile grid in uso è {id}",
+            "disableCustomTileStrategy": "Disabilita la strategia personalizzata per la tile grid",
+            "enableCustomTileStrategy": "Abilita la strategia personalizzata per la tile grid",
```

Comment:
Pay attention to the translation, custom tile grids, not custom strategy. This problem should go away when talking about fetching tile grids instead of enabling/disabling strategy.